metric_learn/mmc.py

```diff
@@ -261,9 +261,9 @@ def _fit_diag(self, X, constraints):
       # search over optimal lambda
       lambd = 1  # initial step-size
       w_tmp = np.maximum(0, w - lambd * step)
-
       obj = np.dot(s_sum, w_tmp) + self.diagonal_c * self._D_objective(X, c, d, w_tmp)
-      obj_previous = obj * 1.1  # just to get the while-loop started
+      assert_all_finite(obj)
+      obj_previous = obj + 1.1  # just to get the while-loop started
```

Comment:
`obj + 1` may be less confusing to readers here.

Reply:
Agreed, done